src/crd-controller/Models/KamusSecret.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using k8s;
+using k8s.Models;
+
+namespace CustomResourceDescriptorController.Models
+{
+    public class KamusSecret : KubernetesObject
+    {
+        public Dictionary<string, string> Data { get; set; }
+        public string Type { get; set; }
+        public V1ObjectMeta Metadata { get; set; }
+        public string ServiceAccount { get; set; }
```

Comment:
Isn't that an issue where the property names are capitalized here and camel cased in the `KamusSecret` k8s object (in the docs)?

Reply:
It worked, I'm not 100% sure why 🤷‍♂️